src/main/java/com/gmail/goosius/siegewar/SiegeWar.java

```diff
@@ -51,42 +53,54 @@ public void onEnable() {
     	
         if (!townyVersionCheck(getTownyVersion())) {
             System.err.println(prefix + "Towny version does not meet required minimum version: " + requiredTownyVersion.toString());
-            onDisable();
+            isError = true;
         } else {
             System.out.println(prefix + "Towny version " + getTownyVersion() + " found.");
         }
         
         if (!Settings.loadSettingsAndLang())
-        	onDisable();
+        	isError = true;
+
+		registerCommands();

```

Comment:
registerCommands() looks to have been indented here.

Reply:
Fixed